java/ql/lib/ext/io.netty.handler.codec.model.yml

```diff
@@ -0,0 +1,58 @@
+      - ["io.netty.handler.codec", "Headers", True, "names", "()", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "add", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addBoolean", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addByte", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addChar", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addDouble", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addFloat", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addInt", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addLong", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addObject", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addShort", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "addTimeMillis", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "set", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setAll", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setBoolean", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setByte", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setChar", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setDouble", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setFloat", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setInt", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setLong", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setObject", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setShort", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
+      - ["io.netty.handler.codec", "Headers", True, "setTimeMillis", "", "", "Argument[-1]", "ReturnValue", "value", "manual"]
```

Comment:
Would it make sense to taint the qualifier from the arguments here as well (i.e. modelling the "setter" logic)?